openapi.yaml

```diff
@@ -3459,6 +3469,14 @@ paths:
         * Entering an empty array (`[]`) blocks all connections (both public and private) to the Managed Database.
         * **Note**: Updates to the `allow_list` may take a short period of time to complete, making this command inappropriate for rapid successive updates to this property.
 
+        All Managed Databases include automatic patch updates. The maintenance window for these updates is configured with the Managed Database `updates` property.
+
+        * By default, the maintenance window is set to start *every week* on *Sunday* at *20:00 UTC* and lasts for 3 hours.
+
+        * If your database cluster is configured with a single node, you will experience downtime during this maintenance window when any updates occur. It's recommended that you adjust this window to match a time that will be the least disruptive for your application and users. You may also want to consider upgrading to a high availability plan to avoid any downtime due to maintenance.
+
+        * Currently, these automatic updates do not include minor release for the database engine software (DBMS). To upgrade to a new minor version (such as upgrading MySQL 8.0.26 to 8.0.27), consider deploying a new Managed Database with your preferred version. Then, once it's deployed, you can [migrate your database](/docs/products/databases/managed-databases/guides/migrate-mysql/).
```

Comment:
I think this is backwards. I think we don't upgrade to a major version. The patch endpoint should upgrade minor and patch versions.

cc @alsaihn